plugins/woocommerce-admin/client/marketplace/components/tabs/tabs.tsx

```diff
@@ -141,12 +107,13 @@ const renderTabs = (
 					key={ tabKey }
 				>
 					{ tabs[ tabKey ]?.title }
-					{ tabs[ tabKey ]?.showUpdateCount &&
-						tabs[ tabKey ]?.updateCount > 0 && (
-							<span className="woocommerce-marketplace__update-count">
-								<span> { tabs[ tabKey ]?.updateCount } </span>
-							</span>
-						) }
+					{ tabs[ tabKey ]?.showUpdateCount && (
+						<span
```

Comment:
We should be showing update counts only if there's a search term:

Designs without a search term:
![image](https://github.com/user-attachments/assets/28c38aa8-1ba0-4f99-ad43-95d81f5ccc76)

With search term:
![image](https://github.com/user-attachments/assets/a4132b49-268b-41ec-8af6-ccb72f466f80)


Implementation:
![image](https://github.com/user-attachments/assets/98472156-6c3a-4a12-b20a-1fdb325a2cf6)

Reply:
Thanks! Nice catch. Addressed in 30cbb80390